tags/tag-operational-resilience/charter.md

```diff
@@ -1 +1,83 @@
+* **Troubleshooting:** The systematic process of identifying, diagnosing, and resolving issues within a distributed system.
+* **Testing:** Verifying the reliability and resilience of applications
+* **Reliability:** The ability of a system to perform its required functions under stated conditions for a specified period of time.
+* **Day 2 Ops:** A focus on the post-deployment activities required to keep a system running smoothly, including maintenance, scaling, updating and upgrading.
+* **Chaos Engineering:** The practice of experimenting on a system to build confidence in its capability to withstand turbulent and unexpected conditions.
+* **Conformance Programs:** Relevant ownership and development of conformance programs to help end users adapt and choose platforms and tools that adhere to established operational resilience standards
+
+### Out of Scope
+
+* This TAG will collaborate with other TAGs on overlapping areas, but will not primarily focus on topics explicitly owned by different TAGs
+* Form an umbrella organization beyond the CNCF
+* Evaluate individual company infrastructures and provide advice
+* Focus outside of cloud native technologies, according to the [CNCF Cloud Native definition](https://github.com/cncf/toc/blob/main/DEFINITION.md)
+
+
+## Deliverables to TOC
+
+The anticipated deliverables for TAG Operational Resilience include:
+
+* **Subprojects:** Ongoing services or programs requiring stewardship from the TAG. Examples include:
+  * Production Readiness Assessments
+  * Resource Optimization and Green Reviews
+* **Initiatives:** Time-bound, objective-focused units of work submitted to the TOC. Examples may include:
+  * Specific assessments for Resource Optimization or Cost Efficiency best practices
+  * Guidelines for performance testing best practices
+* **Whitepapers and Guides:** In-depth documents that provide frameworks and guidelines consistent with advancing practices in the TAG's scope. These serve projects, other TAGs, and the TOC. 
+* **Landscape Analysis:** A curated view of the projects, tools, and standards within the operational resilience ecosystem, similar to the Cloud Native Landscape, to identify gaps and opportunities.
+* **Knowledge Gap Identification:** Reports and presentations to the TOC on identified gaps within the cloud native ecosystem related to operational resilience. 
+* **Suggestions for improvements:** Recommend improvements for CNCF internal processes. Examples include suggestions for:
+    * New educational initiatives, such as for sustainability.
+    * Centralized conformance and supply chain tooling for projects.
+* **Conformance Programs**: Supporting the creation of, and providing oversight for, conformance programs that fall within the TAG's scope. These programs help ensure the quality and resilience of vendor and project software in the CNCF landscape.
```

Comment:
```suggestion
* **Conformance Programs**: Supporting the creation of, and providing oversight for, conformance programs that fall within the TAG's scope. These programs help ensure the quality and resilience projects in the CNCF landscape.
```
As a TAG, the outputs should focus on the projects and adopters of technology, not on vendor specific software. the suggested verbiage will make this more explicit.